plugins/poll/test/javascripts/component/poll-results-ranked-choice-test.js

```diff
@@ -58,4 +58,16 @@ module("Poll | Component | poll-results-ranked-choice", function (hooks) {
       "displays the winner information"
     );
   });
+
+  test("Renders the ranked choice results component without error when outcome data is empty", async function (assert) {
+    this.setProperties({
```

Comment:
you are setting only one property

Reply:
Fair enough, simplified: https://github.com/discourse/discourse/pull/28542/commits/22d26ae16a1e794a90f649ac842697cfbe6047e8